src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -360,6 +370,7 @@ class Storage extends React.Component {
 
   onCreateDisk () {
     const newId = generateUnique('NEW_')
+    const diskInitialSize = 1
```

Comment:
This should be a prop to the component and probably driven by a constant in `CreateNewVm` where the `Storage` component is used.